open-machine-learning-jupyter-book/ml-fundamentals/ml-overview.md

```diff
@@ -0,0 +1,616 @@
+# Machine learning : an overview
```

Comment:
Why do we need to create a new ML overview section? ML content already has an introduction section.